components/ConnectPage/ConnectPage.tsx

```diff
@@ -42,34 +42,54 @@ function ConnectPage() {
   }, [session, address])
 
   return (
-    <div className="flex flex-col mt-3">
-      {address && <NavBar />}
-      <div className="flex flex-col items-center justify-around text-white pt-10 h-[75vh]">
-        <span>
-          {session?.user && <small>Signed in as</small>}
-          <br />
-          {session?.user && <strong>{session?.user.email || session.user.name}</strong>}
-          <a
-            href={session?.user ? "/api/auth/signout" : "/api/auth/signin"}
+    <div className="bg-[url('/leaderboard_background.png')]">
+      <div className="flex flex-col items-center justify-center h-screen gap-y-4 ">
+        {!session?.user && (
+          <Image
+            src="/twitter_log_in.png"
+            alt="Twitter Login"
+            width={400}
+            height={100}
+            loader={customLoader}
             onClick={(e) => {
               e.preventDefault()
-              if (session?.user) {
-                signOut()
-              } else {
-                signIn("twitter")
-              }
+              signIn("twitter")
             }}
-          >
-            <Button onClick={(e) => e.preventDefault()}>
-              {session?.user ? "Sign out" : "Twitter Sign in"}
-            </Button>
-          </a>
-        </span>
-
-        {!address && <ConnectButton />}
+          />
+        )}
+        {session?.user && (
+          <Image
+            src="/twitter_logout.png"
+            alt="Twitter Login"
+            width={400}
+            height={100}
+            loader={customLoader}
+            onClick={(e) => {
+              e.preventDefault()
+              signOut()
+            }}
+          />
```

Comment:
why is this code block duplicated?

Seems like this `<Image />` is being rendered in both cases, right?